src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -631,6 +631,18 @@ public function testMerge()
         $childDefB = $config->registerForAutoconfiguration('BInterface');
         $container->merge($config);
         $this->assertSame(['AInterface' => $childDefA, 'BInterface' => $childDefB], $container->getAutoconfiguredInstanceof());
+
+        $container = new ContainerBuilder();
```

Comment:
I would make this a separate `testMerge*` method instead of having one more separate case in the same test. This is easier to understand the intent of the test or the failure (and corresponds to the best practices of how to use PHPUnit).
Many existing tests in the older Symfony components are mixing many tests in a single method, because the Symfony team was new to PHPUnit when writing them.